packages/ui/product/Product.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import Star from './Star';
+import {
+  Title,
+  Card,
+  Image,
+  Text,
+  Badge,
+  Button,
+  Group,
+  Rating,
+} from '@mantine/core';
+
+const ProductImage = styled.img`
+  height: 200px;
+`;
+
+const CardWrapper = styled.div`
+  margin: 48px auto 0;
+  width: 300px;
+  height: 600px;
+  // display: flex; TO DO - add this when used for the catalog page
+`;
+
+const CardDetails = styled.div`
```

Comment:
Move all styled components to styles.ts

Reply:
done in [e335910](https://github.com/dandalandone/POC-EyCommerce/pull/36/commits/e335910cb23a7361817e14053f9300ccb2fc5d9b)